Zend/zend_compile.h

```diff
@@ -757,6 +759,16 @@ int zend_add_literal(zend_op_array *op_array, zval *zv);
 #define ZEND_FETCH_CLASS_NO_AUTOLOAD 0x80
 #define ZEND_FETCH_CLASS_SILENT      0x0100
 
+/* reserved names */
+#define ZEND_NOT_RESERVED			0
+#define ZEND_RESERVED_STATIC		1
+#define ZEND_RESERVED_ABSTRACT		2
+#define ZEND_RESERVED_FINAL			3
+#define ZEND_RESERVED_PRIVATE		4
+#define ZEND_RESERVED_PROTECTED		5
+#define ZEND_RESERVED_PUBLIC		6
+#define ZEND_RESERVED_CLASS			7
```

Comment:
It could be useful in case one wants to check for a specific identifier to give more specialized err messages, like: https://github.com/php/php-src/pull/1054/files#diff-3a8139128d4026ce0cb0c86beba4e6b9R4511